src/software/amazon/ion/impl/PrivateUtils.java

```diff
@@ -228,10 +162,11 @@ public static SymbolToken newSymbolToken(SymbolTable symtab,
                                              String text)
     {
         // TODO amzn/ion-java#21 symtab should not be null
-        if (text == null || text.length() == 0)
+        if (text == null)
         {
             throw new EmptySymbolException();
```

Comment:
What did you determine the impact to be? Will the new code compile for users who explicitly check for EmptySymbolException?